CIP-1694/README.md

```diff
@@ -0,0 +1,1142 @@
+3. a ratified action is **dropped**
+
+Some actions will be enacted _immediately_ (i.e. at the same epoch boundary they are ratified), others are enacted only on _the following epoch boundary_.
+
+| Governance action type         | Enactment           |
+| :--                            | :--                 |
+| 1. Motion of no-confidence     | Immediate           |
+| 2. New committee/quorum        | Immediate           |
+| 3. Update to the Constitution  | Immediate           |
+| 4. Hard-fork initiation        | Next epoch boundary |
+| 5. Protocol parameter changes  | Next epoch boundary |
+| 6. Treasury withdrawal         | Immediate           |
+| 7. Info                        | Immediate           |
+
+<!-- TODO - break up the protocol parameters into those which effect the header/body validation split (so that some can be enacted immediately)? -->
+
+#### Content
+
+Every governance action will include the following:
+
+* a deposit amount (recorded since the amount of the deposit is an updatable protocol parameter)
+* a reward address to receive the deposit when it is repaid
+* an anchor for any metadata that is needed to justify the action
+* a hash digest value to prevent collisions with competing actions of the same type (as described earlier)
+
+<!-- TODO: Provide a CBOR specification in the annexe for these new on-chain entities -->
+
+In addition, each action will include some elements that are specific to its type:
+
+| Governance action type         | Additional data                                               |
+| :--                            | :--                                                           |
+| 1. Motion of no-confidence     | None                                                          |
+| 2. New committee/quorum        | The set of verification key hash digests and a quorum number  |
+| 3. Update to the Constitution  | A hash digest of the Constitution document                    |
+| 4. Hard-fork initiation        | The new (greater) major protocol version                      |
+| 5. Protocol parameters changes | The changed parameters                                        |
+| 6. Treasury withdrawal         | A map from stake credentials to a positive number of Lovelace |
+| 7. Info                        |                                                               |
+
+Each accepted governance action will be assigned a unique identifier (a.k.a. the **governance action ID**),
+consisting of the transaction hash that created it and the index within the transaction body that points to it.
+
+> ** Note: There can be no duplicate committee members - each pair of credentials in a committee must be unique.
+
+#### Protocol Parameter groups
+
+We have grouped the protocol parameter changes by type.
+This allows different thresholds to be set for each group, and also supports separate votes.
+DReps may choose to abstain to vote on parameter changes that are outside of their field of expertise.
```

Comment:
If I delegate to a dRep who voluntarily limits their “field of expertise”, my stake will only be used in that field, but not in any other field that I might also find important. Worse, in a given epoch, there might be two governance actions pending, I find both important, but the dReps choose to abstain in one or the other.

If such a grouping of fields of expertise is done, the dReps should register for groups explicitly and I should be able to choose different dReps for each group. And the concept of “field of expertise” should be extended to the other governance action types.

Reply:
It's entirely up to the DRep. Have different DReps for different actions is something we discussed for sure, but it complicates the design (both logistically and from the engineering standpoint) too much, in our opinion, to put it in the minimal viable version. A DRep who wants to focus narrowly is better suited to joining up with other experts who have a well rounded expertise. The could collaborate off chain, or use scripts.

Reply:
> A DRep who wants to focus narrowly is better suited to joining up with other experts who have a well rounded expertise. The could collaborate off chain, or use scripts.

Shouldn't the CIP then encourage such cooperations instead of encouraging to abstain outside the field of expertise?
As long as I can only delegate to one dRep, I would definitely want them to vote on all proposals ever and only abstain if there are real good and specific reasons for it.

Reply:
I went to edit this just now, and I found that this isn't even a good spot for this discussion. I'll remove the prose for now and try to incorporate what we talked about here somewhere more appropriate.